lib/db_utils.go

```diff
@@ -4216,6 +4216,25 @@ func DecodeUint64(scoreBytes []byte) uint64 {
 	return binary.BigEndian.Uint64(scoreBytes)
 }
 
+func EncodeUint16(num uint16) []byte {
```

Comment:
I added EncodeUint16 instead of EncodeUint8 since binary.BigEndian only supports Uint16 not Uint8. Still saves us quite a few bytes. One wasted byte when encoding Uint8s as Uint16s. IMO fine for now since binary.BigEndian handles all the heavy lifting for us.

Reply:
I figured out EncodeUint8. It was easy enough.